models/nc-news.models.js

```diff
@@ -53,7 +58,7 @@ exports.fetchArticles = (articleId, topicQuery) => {
       }
 
       queryStr += ` GROUP BY articles.article_id 
-      ORDER BY created_at DESC;`;
+      ORDER BY ${sortBy} ${orderBy};`;
```

Comment:
Both sortBy and orderBy need to have a list of values whitelisted so there is no risk of just any code being ran here.
You could for example do a validSortBy array and check against that and do an if/else if statement for asc/desc order